HeimdallrTests/Core/HeimdallrSpec.swift

```diff
@@ -725,6 +725,34 @@ class HeimdallrSpec: QuickSpec {
                         expect(result?.error?.code).to(equal(OAuthErrorInvalidClient))
                     }
                 }
+
+                context("when issueing multiple requests at the same time") {
+                    it("only issues one refresh token call") {
```

Comment:
Add an expectation to the HTTP stub verifying that the first completion block got called before the second HTTP request is received.